contracts/test/DaiEchidnaTest.sol

```diff
@@ -50,6 +58,8 @@ contract DaiEchidnaTest {
 
     /// @dev Test that supply and balance hold on transfer.
     function transfer(uint256 wad) public {
+        wad = wad % uint256(-1);
```

Comment:
Similar comment--you don't want to try to transfer more than this address holds.

Reply:
Done.

Reply:
same as above for `transfer` test.

Reply:
Done.